lib/bigtable/graph/GraphAccess.js

```diff
@@ -23,9 +23,9 @@ class GraphAccessSingle {
         this.translator = await this.yildiz.getTranslator();
     }
 
-    async setLastAccess(values) {
+    setLastAccess(values) {
```

Comment:
setLastAccessFireAndForget (dont make this async, use a simple .catch(() => {/* empty */});